warehouse/query-core/src/main/java/datawave/query/predicate/AncestorEventDataFilter.java

```diff
@@ -24,10 +26,21 @@ public class AncestorEventDataFilter extends EventDataQueryExpressionFilter {
      * @param metadata
      *            type metadata
      */
+    @Deprecated
```

Comment:
I would just get rid of this constructor.

Reply:
See other comment

Reply:
See other comment